docs/BuildFromSource.md

```diff
@@ -129,6 +131,8 @@ On macOS/Linux:
 ./build.sh
 ```
 
+By default, only the C# projects are built. Some sub-areas have additional requirements, such as needing Yarn to be installed, before being able to build successfully.
```

Comment:
> By default, only the C# projects are built.

Would it be better to say that by default we build all of the managed projects - and some of those have dependencies on Node/Yarn. 

Then say what to do to skip those.